manifests/init.pp

```diff
@@ -74,17 +74,22 @@
       $_tz = split($timezone, '/')
       $area = $_tz[0]
       $zone = $_tz[1]
-      exec { 'update_debconf area':
-        command => "echo tzdata tzdata/Areas select ${area} | debconf-set-selections",
-        unless  => "debconf-get-selections |grep -q -E \"^tzdata\\s+tzdata/Areas\\s+select\\s+${area}\"",
-        path    => $facts['path'],
-      }
-      exec { 'update_debconf zone':
-        command => "echo tzdata tzdata/Zones/${area} select ${timezone} | debconf-set-selections",
-        unless  => "debconf-get-selections |grep -E \"^tzdata\\s+tzdata/Zones/${area}\\s+select\\s+${zone}\"",
-        path    => $facts['path'],
+
+      debconf {
+        'tzdata/Areas':
+          package => 'tzdata',
+          item    => 'tzdata/Areas',
+          type    => 'select',
+          value   => $area;
+        "tzdata/Zones/${area}":
+          package => 'tzdata',
+          item    => "tzdata/Zones/${area}",
+          type    => 'select',
+          value   => $timezone;
```

Comment:
I think it should be $zone instead of $timezone.

With $timezone i get
`tzdata	tzdata/Zones/Europe	select	Europe/Paris`
instead of 
`tzdata	tzdata/Zones/Europe	select	Paris`
which is the value set using `dpkg-reconfigure tzdata`

Reply:
Thanks.